client/client_test.go

```diff
@@ -1255,7 +1172,6 @@ func TestClient_UpdateNodeFromFingerprintKeepsConfig(t *testing.T) {
 	client, cleanup := TestClient(t, nil)
 	defer cleanup()
 	// capture the platform fingerprinted device name for the next test
```

Comment:
This comment line is orphaned now, I think? It looks like it refers to the moved section.